packages/trader/src/Modules/Trading/Components/Form/ContractType/ContractTypeInfo/contract-type-info.tsx

```diff
@@ -36,6 +37,7 @@ const Info = ({ handleSelect, item, list }: TInfo) => {
     const is_glossary_tab_selected = selected_tab === TABS.GLOSSARY;
     const width = isMobile() ? '328' : '528';
     const scroll_bar_height = has_toggle_buttons ? '464px' : '560px';
+    const { is_vanilla_fx } = useTraderStore();
```

Comment:
Got question: if we use `useTraderStore()`, then do we need to cover component with `observer`?

Reply:
good catch )

Reply:
Actually, observer is not really useful here anyhow, as there's no way the variable (symbol) can change without closing/reloading the info component anyways. But it seems from Mobx docs that if it's not used, it would anyhow affect performance, so I added it just in case.